huaweicloud/config/config.go

```diff
@@ -426,6 +518,15 @@ func (c *Config) NewServiceClient(srv, region string) (*golangsdk.ServiceClient,
 		return nil, fmt.Errorf("service type %s is invalid or not supportted", srv)
 	}
 
+	if c.SecurityKeyExpiresAt != "" {
+		timeNow := time.Now().Unix()
+		expairesAt, _ := time.Parse(time.RFC3339, c.SecurityKeyExpiresAt)
+		expairesAtInt := expairesAt.Unix()
+		if timeNow+keyExpiresDuration > expairesAtInt {
```

Comment:
suggest to ues `time.Since(xxx).Seconds() > expairesAtInt`

Reply:
as we discussed, keep it as is